src/core/style/core.scss

```diff
@@ -0,0 +1,12 @@
+// Core styles that can be used to apply material design treatments to any element.
```

Comment:
This is good for now, but FYI I'm going to move it to `src/main.scss` once my outstanding PR lands.